app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt

```diff
@@ -249,7 +249,7 @@ fun PlayerScreen(
                                 }
                             }
                         },
-                    tint = if (!listenLiked) Color.Red else Color.Black
+                    tint = MaterialTheme.colorScheme.onSurface
```

Comment:
I think the if else conditions should be retained here and the color should be red when liked and in the else condition we can have the onsurface color